HF-122: Framework integration guides for React, Angular, Vue, Svelte (#1653)
handsontable/hyperformula: docs/guide/integration-with-angular.md

# Integration with Angular

> The HyperFormula API is identical in an Angular app and in plain JavaScript. What changes is where the engine lives (typically an injectable service), how it is cleaned up, and how you bridge its values into the change-detection cycle.

Reviewer: ```suggestion
The HyperFormula API is identical in an Angular app and in plain JavaScript. This guide demonstrates how HyperFormula is integrated with an Angular app (typically as an injectable service), how it is cleaned up, and how you bridge its values into the change-detection cycle.
```

I don't like the phrase "What changes is..."

Author: Adopted verbatim in [`b080681`](https://github.com/handsontable/hyperformula/pull/1653/commits/b080681dc). Better wording — thanks.
